### ISSUE TITLE
Zoo Keras BatchNormalization layer needs to support other dimension input besides 4D input

### ISSUE THREAD

**Issue by jenniew** [intel-analytics/analytics-zoo#1222]
Current keras BatchNormalization can only accept 4D input  supposed to be the convolution output of image, but BatchNormalization needs to support other dimension input for broader usage.
